src/emqx_cm.erl

```diff
@@ -72,19 +73,26 @@
         ]).
 
 %% Internal export
--export([stats_fun/0, clean_down/1]).
+-export([ stats_fun/0
+        , clean_down/1
+        , increment_connected_client_count/1
+        , decrement_connected_client_count/1
+        , get_connected_client_count/0
+        ]).
 
 -type(chan_pid() :: pid()).
 
 %% Tables for channel management.
 -define(CHAN_TAB, emqx_channel).
 -define(CHAN_CONN_TAB, emqx_channel_conn).
 -define(CHAN_INFO_TAB, emqx_channel_info).
+-define(CONN_CLIENT_TAB, connected_client_counter).
```

Comment:
```suggestion
-define(CONN_CLIENT_TAB, emqx_channel_connected).
```
```suggestion
-define(CONN_CLIENT_TAB, connected_client_counter).
```

Reply:
or maybe `emqx_channel_live` to avoid being prefixed by the other table `emqx_channel_conn`